angular.json

```diff
@@ -349,43 +349,6 @@
         }
       }
```

Comment:
We should add one line to our deprecation docs and mention about the removal of incubator lib

Reply:
Done